packages/lib/src/components/Giftcard/Giftcard.tsx

```diff
@@ -4,9 +4,10 @@ import GiftcardComponent from './components/GiftcardComponent';
 import CoreProvider from '../../core/Context/CoreProvider';
 import getImage from '../../utils/get-image';
 import PayButton from '../internal/PayButton';
+import { GIFT_CARD } from '../internal/SecuredFields/lib/configuration/constants';
 
 export class GiftcardElement extends UIElement {
-    public static type = 'giftcard';
+    public static type = GIFT_CARD;
```

Comment:
Are we sure we want to use the Secured Fields type for the payment method type? They seem different things (even if they share the same value).

Reply:
I'm using an existing constant so we don't have loads of 'giftcard' strings lying around.
But I can change it back

Reply:
Let's change this one back but keep the others, so it won't break if at some point the other value changes. Rest looks good! 👍

Reply:
ok